### ISSUE TITLE
Links look like regular text in High Contrast Black Theme

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1011]
**Describe the bug**
In the Fiori 3 'high contrast black' theme - Links look like regular text.

> What version of the Fundamental Library Styles are you using?
v0.9.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/link.html
2. It is not possible to visually distinguish links from regular text because of loss of color.

**Expected behavior**
Users should be able to visually distinguish between a link and normal text in a Fiori3 high-contrast black theme. Either use a different color (yellow) or use some type of text-decoration. Using a different color is recommended.

**Screenshots**
Links in the normal theme have a blue color
![image](https://user-images.githubusercontent.com/43764832/81754583-69766600-946b-11ea-8a84-b85ad2349c24.png)

Links in the Fiori 3 HCB theme have white color which is the same as normal text.
![image](https://user-images.githubusercontent.com/43764832/81754544-4fd51e80-946b-11ea-9887-bb622db4793c.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by droshev** [comment]
This has been addressed as part of the delta theming
